FutureCollars/src/main/java/lesson5task4/Employee.java

```diff
@@ -0,0 +1,57 @@
+package lesson5task4;
+
+import java.math.BigDecimal;
+
+public class Employee {
+    private String name;
+    private String surname;
+    private int age;
+    private BigDecimal baseSalary;
+    private BigDecimal bonus;
+
+    public Employee(String name, String surname) {
+        this.name = name;
+        this.surname = surname;
+    }
+
+    public Employee(String name, String surname, int age) {
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+    }
+
+    public Employee(String name, String surname, int age, BigDecimal baseSalary) {
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+        this.baseSalary = baseSalary;
+    }
```

Comment:
W tych konstruktorach nie zawsze ustawiasz wartość bonus i baseSalary. Jeżeli ktoś stworzy za ich pomocą obiekt Employee i następnie użyje metody getTotalSalary to dostanie NullPointerException. Pomyśl nad przypisaniem tym polom wartości 0, jeżeli wartość nie jest podana :)